std_msgs/Header header
common/Uint8TimeStamped adcu_cld_ctrlative
common/Uint8TimeStamped adcu_cld_prkenable
common/Uint8TimeStamped adcu_cld_throtmode
common/Uint8TimeStamped adcu_cld_tgtgear
common/FloatTimeStamped adcu_cld_tgtthrotval
common/FloatTimeStamped adcu_cld_tgtbrkval
common/FloatTimeStamped adcu_cld_tgtstrangle

